code/modules/transport/pods/vehicle.dm

```diff
@@ -112,6 +113,17 @@
 	attackby(obj/item/W, mob/living/user)
 		user.lastattacked = src
 		if (health < maxhealth && isweldingtool(W))
+			var/turf/T = get_turf(src)
+			if(T.active_liquid)
+				if(T.active_liquid.my_depth_level >= 3 && T.active_liquid.group.reagents.get_reagent_amount("tene")) //SO MANY PERIODS
+					boutput(user, "<span class='alert'>The damaged parts are saturated with fluid. You need to move somewhere drier.</span>")
```

Comment:
This should probably have the same message as the other one.

Reply:
Functionally it's a little different, so I'm personally inclined to keep the differing message.